app/assets/stylesheets/_mixins.scss

```diff
@@ -4,3 +4,12 @@
   letter-spacing: $letter-spacing-header;
   text-transform: uppercase;
 }
+
+@mixin block-link {
```

Comment:
> This expands the link to be the size of the container making it feel like you're clicking the entire container.

Ah, so it makes it so you can click on even the whitespace in the container and it works? And without this, clicking on whitespace won't work? (But clicking on "real words" will?)